### PR TITLE
fix: change already friends status code from 403 to 409

### PR DESCRIPTION
**Pull request by faitti**
409 Conflict fits this situation better than 403 Forbidden

### CODE REVIEW

**Review by DrVilepis — APPROVED** [review]
very good

File: src/error.rs [hunk 1]
```diff
@@ -68,11 +68,11 @@ impl ResponseError for TimeError {
             | TimeError::BadLeaderboardName => StatusCode::BAD_REQUEST,
             TimeError::CurrentUser
             | TimeError::UserExists
-            | TimeError::AlreadyFriends
             | TimeError::LeaderboardExists
```
Comment (Eldemarkki) [inline src/error.rs:71]: I think the LeaderboardExists should also be a 409 Conflict

**Review by Eldemarkki — CHANGES_REQUESTED** [review]

File: src/error.rs [hunk 2]
```diff
@@ -68,11 +68,11 @@ impl ResponseError for TimeError {
...
             | TimeError::AlreadyMember
```
Comment (Eldemarkki) [inline src/error.rs:72]: AlreadyMember should also be a 409 Conflict

**Review by Eldemarkki — CHANGES_REQUESTED** [review]

**Comment by faitti** [comment]
I also changed LeaderboardExists, AlreadyMember and UserExists to 409

File: src/error.rs [hunk 3]
```diff
@@ -66,13 +66,13 @@ impl ResponseError for TimeError {
             | TimeError::InvalidLength(_)
             | TimeError::BadId
             | TimeError::BadLeaderboardName => StatusCode::BAD_REQUEST,
-            TimeError::CurrentUser
-            | TimeError::UserExists
-            | TimeError::AlreadyFriends
+            TimeError::CurrentUser | TimeError::NotMember | TimeError::LastAdmin => {
```
Comment (lajp) [inline src/error.rs:69]: Does this need to be a block?

Comment (faitti) [reply src/error.rs:69]: cargo fmt turned it into a one so i guess it has to be

Comment (lajp) [reply src/error.rs:69]: so be it then

**Review by lajp — APPROVED** [review]
LGTM

**Comment by lajp** [comment]
Thx for ur contribution @faitti! We wish to see you here again :)

**Pull request merged**